test/test_autograd.py

```diff
@@ -2545,6 +2545,65 @@ def run_test(upper, dims):
         for upper, dims in product([True, False], [(3, 3), (5, 3, 3), (4, 3, 2, 2)]):
             run_test(upper, dims)
 
+    @skipIfNoLapack
```

Comment:
If this is extra slow, you can add a `@slowTest` to avoid running it on all PRs and just on master slow test.

Reply:
Sure, will do.